src/commands/moderation.rs

```diff
@@ -196,14 +199,15 @@ pub async fn unmute(
 #[poise::command(slash_command)]
 pub async fn mute(
     ctx: Context<'_>,
-    #[description = "The member to mute"] mut member: Member,
+    #[description = "The member to mute"] member: UserId,
     #[description = "Seconds"] seconds: Option<i64>,
     #[description = "Minutes"] minutes: Option<i64>,
     #[description = "Hours"] hours: Option<i64>,
     #[description = "Days"] days: Option<i64>,
     #[description = "Months"] months: Option<i64>,
     #[description = "The reason of the mute"] reason: String,
 ) -> Result<(), Error> {
+    let member = to_user!(member, ctx);
```

Comment:
Yes, the parameter should no longer be called `member` since it works on any user now.
I do not necessarily think the returning early part makes any difference. Poise used to convert `UserId` to `User` for us before even calling the command function, but now we are just doing it explicitly to avoid the serenity bug until the fix for it gets released.

When the fix does get released, we can just remove the macro and make the functions accept `User` again (better autocomplete).